rst/telemetry.rst

```diff
@@ -2,22 +2,27 @@
 
 Telemetry
 =========
-Telemetry provides further insight into a match.
+Telemetry provides further insight into a match. This is where you will find the :ref:`telemetry-events` that occur throughout the match. In order to get the telemetry data, you will first need to get the link for it from within the match object. Detailed steps can be found below.
+
+.. role:: raw-html(raw)
+   :format: html
+
+Data dictionaries and enums can be found :raw-html:`<a class="tosAlert">here</a>`.
 
 To retrieve Telemetry Data
 ---------------------------
-Telemetry data is compressed using gzip. Many browsers and libraries support this without any extra work being necessary, but clients using the API should specify that they accept gzip compressed responses. Make sure that you pass the "Accept-Encoding: gzip" header.
-
-You will need to start by pulling a match from the matches endpoint. Remember to replace "some-region" and "matchId" with the appropriate region and matchId respectively.
+You will need to start by pulling a match from the matches endpoint. Remember to replace $platform and "matchId" with the appropriate platform and matchId respectively.
 
 Note: In order to get a match object, you will need to know its ID. You can find match IDs inside of player objects returned from the players endpoint. For more information about the players endpoint check out :ref:`players`. There is also a tutorial that may be helpful on the :ref:`getting-started` page.
 
+Your request should look something like the following:
+
 .. code-block:: shell
 
-  curl "https://api.pubg.com/shards/some-region/matches/matchId" \
+  curl "https://api.pubg.com/shards/$platformn/matches/$matchId" \
   -H "Accept: application/vnd.api+json"
 
-Next, look for an assets reference in the match relationships object, which points to (contains the ID of) the full telemetry object within the included array. Check for the following in the response::
+Next, you will need to find an "assets" reference in the match "relationships" object. This contains the ID of the full telemetry object. You will need this ID to locate the link for the telemetry file. Check for the following in the response::

```

Comment:
Should we specifically state that they will need to find the asset reference within the match file that they just downloaded? Seems obvious to me, but it could confuse some.

Maybe also clarify that the full path to the reference is `data.relationships.assets.data.id`, and that they have to use that to find the asset object in the included array where the path to the URL is `attributes.URL`

Reply:
Nevermind the second part of this. I see it below. 🤦‍♂

Reply:
Eh, I see the other part now too. Maybe just disregard entirely. 😅